server.js

```diff
@@ -28,7 +28,10 @@ var limiter = RateLimit({
 });
 
 app.use(compression());
-app.use(express.static(path.resolve(__dirname, `public`)));
+app.use(express.static(path.resolve(__dirname, `public`), {
+  maxAge: '1m', // expire cache in 1 month
+  immutable: true
```

Comment:
Are you sure we should add this directive? It seems like clients that support it wouldn't even bother to check for updates on a force refresh?

Reply:
do you think static assets would ever need to be updated though? We would just change the name of the asset wouldn't we?

Reply:
are we serving any compiled JS or CSS bundles?

Reply:
(and yeah, if we're prefixing the filename with a hash then we don't have a problem, but that assumes we're already doing that, so... are we?)

Reply:
good catch: https://github.com/mozilla/mozillafestival.org/blob/master/webpack.config.js#L21, I'll remove it